Update to turf 2.0.0?

I clone the repo and ran npm install the first time and received the following error message: 

`
	npm ERR! EEXIST, open '/home/skors/.npm/5551a734-e-skors-npm-rx-2-4-9-package-tgz.lock'
	File exists: /home/skors/.npm/5551a734-e-skors-npm-rx-2-4-9-package-tgz.lock
	Move it away, and try again. 
	npm ERR! System Linux 3.13.0-49-generic
	npm ERR! command "/usr/bin/node" "/usr/bin/npm" "install"
	npm ERR! cwd /home/skors/gis/turf-node-example
	npm ERR! node -v v0.10.38
	npm ERR! npm -v 1.4.28
	npm ERR! path /home/skors/.npm/5551a734-e-skors-npm-rx-2-4-9-package-tgz.lock
	npm ERR! code EEXIST
	npm ERR! errno 47
	npm ERR! not ok code 0
`
I went into the path specified in the error message and there was no file with that name.

I had no idea what to do next. Since turf was updated to 2.0.0 since the last release of this, I thought the new turf could maybe fix it. So I changed turf's requirement in package.json to 2.0.0 

rm -rf'ed node_module folder and ran npm install again, I received no errors, and was able to run this module with no problems. 

Since the error message referred to a locked npm file and nothing to do with turf, I don't know whether the problem was caused by an outdated turf module.

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   "author": "Tom MacWright",
   "license": "ISC",
   "dependencies": {
-    "turf": "^1.3.5"
+    "turf": "^2.0.0"
   },
   "repository": {
     "type": "git",

